tests/format/angular/control-flow/for.html

```diff
@@ -52,3 +52,7 @@
 
   <div *ngFor="item of items; track item"></div>
 </div>
+
+<div>
+  @for (item of items; let i = $index; track block) {}
```

Comment:
I don't know much about angular syntax, but we format `@for` and *nfFor` exactly the same, so please add `<div *ngFor="item of items; let i = $index; track block"></div>` just like the `<div>` above. We can compare them if we change in future.

Reply:
[ea69eac](https://github.com/prettier/prettier/pull/15887/commits/ea69eacb9beb76e5de0345fae0b349bc773d9a59)